libnavui-util/src/main/java/com/mapbox/navigation/ui/utils/internal/SvgUtil.kt

```diff
@@ -8,62 +8,65 @@ import java.io.ByteArrayInputStream
 
 object SvgUtil {
 
-    private const val CSS_RULES =
-        "text { font-family: Arial, Helvetica, sans-serif; font-size: 0.8em}"
-
     @JvmStatic
-    fun renderAsBitmapWithHeight(stream: ByteArrayInputStream, desiredHeight: Int): Bitmap? {
-        try {
+    fun renderAsBitmapWithHeight(
```

Comment:
rendering stream to bitmap might be pretty heavy, should be moved to a callback-based(suspend) approach and moved to a background thread? Or mark method(s) as `@WorkerThread`?